### ISSUE TITLE
[DRIVER3D_MODEL] replacement bug + documentation update request

### ISSUE THREAD

**Issue by PlasticTangerine** [ac-custom-shaders-patch/acc-extension-config#134]
**Bug:**
`DRIVER3D_HIDE_OBJECT` does not work when replacing a driver with `ext_config`.

**Steps to reproduce:**
Add the following lines to `ks_abarth_585ss.ini`:
```INI
[DRIVER3D_MODEL]
; this section allows to replace driver model, could be handy in car skin config
NAME = driver_no_HANS                    ; replace driver model in “driver3d.ini”
DRIVER3D_HIDE_OBJECT = DRIVER:HELMET_SUB0 ; replace meshes to hide in “driver3d.ini”
```
Go in game and go in first person.  Adjust the onboard camera so it looks at the driver's head.  The helmet mesh should be hidden.  It is not hidden.

Tested on CSP versions 0.1.74 and 0.1.75

**Documentation update request:**
On this page: https://github.com/ac-custom-shaders-patch/acc-extension-config/wiki/Cars-%E2%80%93-General-options
Current documentation:
```INI
[DRIVER3D_MODEL]
; this section allows to replace driver model, could be handy in car skin config
NAME = driver.kn5                    ; replace driver model in “driver3d.ini”
DRIVER3D_HIDE_OBJECT = mesh1, mesh2  ; replace meshes to hide in “driver3d.ini”
```
Currently it says driver<b>.kn5</b>, yet if you add .kn5, it changes the driver to `driver_no_hans.kn5` regardless which made me think the feature was broken.  If you don't add .kn5 it works correctly.

Proposed update:
```INI
[DRIVER3D_MODEL]
; this section allows to replace driver model, could be handy in car skin config
NAME = driver                    ; replace driver model in “driver3d.ini”
DRIVER3D_HIDE_OBJECT = mesh1, mesh2  ; replace meshes to hide in “driver3d.ini”
```

**Comment by Trava715** [comment]
Documentation has been updated by @leBluem. As for the other issue, I'm fairly sure it only sets meshes to be hidden in first person cam (which is the default driver3d.ini behavior). If you want to hide them from the model entirely, you have to use KN5 replacement with the HIDE line pointing to your chosen driver model IIRC.

**Comment by gro-ove** [comment]
Thanks for the report! Sorry for the inconvenience, there was another mistake in docs, `DRIVER3D_HIDE_OBJECT_...` is a list.

**Comment by gro-ove** [comment]
Sorry for misleading, would this work?

```ini
[DRIVER3D_HIDE_OBJECT_...]
NAME = …
```

**Comment by PlasticTangerine** [comment]
That does seem to work.  Making the correct code look something like:
```INI
[DRIVER3D_MODEL]
NAME = driver_no_HANS

[DRIVER3D_HIDE_OBJECT_...]
NAME = DRIVER:HELMET_SUB0, DRIVER:HELMET_SUB1, DRIVER:HELMET_GLASS_SUB0, DRIVER:HELMET_GLASS_SUB1, DRIVER:HELMET_plastic, DRIVER:GEO_Driver_FACE
```
Documentation will require an update.


However, I did encounter another problem where cars with VAO patches will still use the VAO on the original driver.  This creates some *interesting...* results on swapped drivers with no helmet for example (`driver_asian` in `ks_audi_r8_plus`): 
![image](https://user-images.githubusercontent.com/74761423/136866054-fec0e100-596c-4608-94ed-bf5ba35e2f89.png)

**Comment by gro-ove** [comment]
Oh, right, that’s a tricky one… I’ll think about a good way to fix it, I guess it would require to rebake driver AO somewhere next to ext_config 🤔

**Comment by Marocco2** [comment]
Still present in 0.1.76?

**Comment by PlasticTangerine** [comment]
> Still present in 0.1.76?

Yes 
![image](https://user-images.githubusercontent.com/74761423/149243479-2575f2bc-2f7a-4cbc-9187-884b15507a32.png)

**Comment by PlasticTangerine** [comment]
Issue resolved in version 0.1.77
